src/code.sh

```
@@ -79,7 +79,7 @@ _generate_region_vcfs() {
             -r "$chromPos" -a FORMAT/AD,FORMAT/DP -Ou | \
             bcftools call -mv -Oz -o "$output_vcf"
 
-        bcftools view -i "REF=='${knownRef}' && ALT=='${knownAlt}'" -Oz -o "${output_vcf}_filtered.vcf.gz" "$output_vcf"
+        bcftools view -i "REF=='${knownRef}' && ALT=='${knownAlt}' && FORMAT/DP>${read_depth_threshold}" -Oz -o "${output_vcf}_filtered.vcf.gz" "$output_vcf"
```

Shellcheck (0.10.0)

[warning] read_depth_threshold is referenced but not assigned.

(SC2154)